docs/repository/repository-github-apps.md

```diff
@@ -45,7 +45,7 @@ Once the GitHub Apps connection is suspended, KintoHub will lose access to the r
 6. Scroll down to the **Danger Zone** section
 7. Click on the **Suspend** button
 
-![Suspend](/img/anatomy/Suspend.png)
+![Suspend](/img/anatomy/suspend-me-please.png)
```

Comment:
action - sorry, it can be suspend! I was giving an example of image with spaces.

if you name it suspend then its `suspend.png`

if you name it Cool Image Example it should be `cool-image-example.png`. It was just giving an example with spaces, but in this case the image name should just suspend with all lowercase.

Reply:
Ah, that's ok. Earlier I had made a mistake by saving an img name in uppercase and defining it in lowercase.

So, it is better to have a common template to avoid such mistakes in the future.